### PR TITLE
Enable StyleCI

### PR DESCRIPTION
**Pull request by lucasmichot**
Enable StyleCI.

---

@aidantwoods, a free account needs to be created at https://styleci.io/ so that `https://github.com/aidantwoods/SecureHeaders` repository can be checked

### CODE REVIEW

File: .styleci.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+preset: psr2
+
+enabled:
+  - short_array_syntax
```
Comment (aidantwoods) [inline .styleci.yml]: I prefer short syntax too myself – PHP 5.3 isn't [too fond](http://php.net/manual/en/migration54.new-features.php) of it though 😜

Comment (lucasmichot) [reply .styleci.yml]: Good catch @aidantwoods , changed that in https://github.com/aidantwoods/SecureHeaders/pull/9/commits/159578149bc8b884898642ec9837292983adb035

**Comment by lucasmichot** [comment]
Seems you've not enabled StyleCI yet @aidantwoods

**Comment by lucasmichot** [comment]
I suppose the branch `analysis-87AMvj` can now be deleted @aidantwoods ?

**Pull request merged**